### ISSUE TITLE
Is it possible to make OLA as a DES?

### ISSUE THREAD

**Issue by jayahm** [scikit-learn-contrib/DESlib#213]
Hi,

It is well known that OLA is a DCS method.

But, is it possible using the library to select the most competence subset of classfiers based on a certain threshold sung OLA?

**Comment by Menelau** [comment]
Hello,

You can set **selection_method='all'** in order to select all base classifiers that obtained the highest competence level instead of randomly picking one among the most competent ones. However, you cannot do that based on a certain threshold. 

If you want to have such functionality, you can look at the DES-P method, which selects all base classifiers according to a certain threshold, which is the performance of a random classifier (RC) in this case (1/n_classes). You can modify this code to allow the user to define a selection threshold instead of always use the RC performance.

**Comment by jayahm** [comment]
Is there any way that I can do in such a way the top N classifiers with highest local accuracy be selected?

Can I modify DES-KNN for this purpose?

**Comment by Menelau** [comment]
The current implementation of DES-KNN you can set the number of classifiers (percentage) to be selected according to accuracy(pct_accuracy, however it is expecting at least one classifier selected according to diversity (**pct_diversity** hyperparameter)

**Comment by jayahm** [comment]
I see. Thank you very much.
